### DISCUSSION TITLE
Script available SMA Meter autostart heating  ?

### DISCUSSION THREAD

**Discussion by dav2-tech** [Yogui79/IntexPureSpa#55] [Q&A]
Hello , is there somewere a script available which starts the heating when the PV Power is high enough as you can see in the snackit ? 

Any  input is highly appreciate ..  

![image](https://user-images.githubusercontent.com/60895099/175938340-79c667e3-1b6b-4017-914a-350ad41a3783.png)

**Comment by LEECHER1** [comment]
I think it easy to make it.
But i have't a PV.

You make a new automatic.

Tricker is the sensor "psupply" when it over 340 W than turn on the Switch
Intex heater. Maybe you must set the Temperature higher as now or set the
Temperature only to 40°c and the heater is always on/standby.



dav2-tech ***@***.***> schrieb am Mo., 27. Juni 2022, 14:14:
